tequilapi/endpoints/service.go

```diff
@@ -189,18 +229,22 @@ func (se *ServiceEndpoint) ServiceStart(resp http.ResponseWriter, req *http.Requ
 //     schema:
 //       "$ref": "#/definitions/ErrorMessageDTO"
 func (se *ServiceEndpoint) ServiceStop(resp http.ResponseWriter, _ *http.Request, params httprouter.Params) {
-	if params.ByName("id") != "6ba7b810-9dad-11d1-80b4-00c04fd430c8" && params.ByName("id") != "7037d6f6-1b92-4513-8cb4-8fd1a944b996" {
-		utils.SendErrorMessage(resp, "Service not found", http.StatusNotFound)
-		return
+	for _, instance := range se.serviceManager.List() {
```

Comment:
this lookup could also be made easier, if we'd have a map instead of a list there :)

Reply:
And I suggest to have have helper functions on `Service.Pool` side. This would leave `ServiceEndpoint` thin -  thin controller and fat Service from MVC principle